From 86e690a46b4a6c2f4b53937567e8379cf9f9c4d4 Mon Sep 17 00:00:00 2001
Message-Id: <86e690a46b4a6c2f4b53937567e8379cf9f9c4d4.1476965418.git.keller@fs-net.de>
In-Reply-To: <1bbe4f4c985819c8f8ead54288ca1cc8a4b7f51f.1476965418.git.keller@fs-net.de>
References: <1bbe4f4c985819c8f8ead54288ca1cc8a4b7f51f.1476965418.git.keller@fs-net.de>
From: Hartmut Keller <keller@fs-net.de>
Date: Wed, 19 Oct 2016 16:00:57 +0200
Subject: [PATCH 08/12] Let weston exit on unknown command line options

Do not silently ignore unknown options, issue help text and exit instead.
---
 src/weston-launch.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/weston-launch.c b/src/weston-launch.c
index b8dfb17..922c23b 100644
--- a/src/weston-launch.c
+++ b/src/weston-launch.c
@@ -701,6 +701,7 @@ main(int argc, char *argv[])
 			wl.verbose = 1;
 			break;
 		case 'h':
+		default:
 			help("weston-launch");
 			exit(EXIT_FAILURE);
 		}
-- 
1.7.4.4

